docs/basics.rst

```diff
@@ -315,6 +315,7 @@ The user can provide these additional parameters:
 
     - ``--dev`` — Install both ``develop`` and ``default`` packages from ``Pipfile``.
     - ``--system`` — Use the system ``pip`` command rather than the one from your virtualenv.
+    - ``--deploy --system`` — The workflow is using pipenv for package loc in local development environment for having a Pipfile and Pipfile.locl file. Deployment can be done using the command. The ``--deploy`` makes sure the packages are properly locked in Pipfile.lock.
```

Comment:
Hmmm, I prefer to mention `--deploy` option only, not a combination, to make them consistent.